packages/ui/locales/en/views.json

```
@@ -462,5 +464,78 @@
     "create": "Create webhook",
     "edit": "Edit webhook",
     "delete": "Delete webhook"
+  },
+  "userManagement": {
+    "addNewUser": "Add a new user",
+    "userId": "User ID",
+    "userIdHint": "User ID cannot be changed once created",
+    "enterUsername": "Enter user name",
```

I'd argue that it's `username`, not `user name`
```suggestion
    "enterUsername": "Enter username",
```